### PR TITLE
Restrict max content width

### LINKED ISSUE #2680: Restrict max-width of content
There's a few pages that don't look too good if run on a maximized browser with lots of pixels.

E.g:

add-ons_for_firefox

Would this work better if constrained? E.g: here's an example using 1400px:

add-ons_for_firefox

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #2680

Before:

screen shot 2017-09-11 at 18 33 20

After: 

screen shot 2017-09-11 at 18 29 59

### CODE REVIEW

File: src/amo/components/App/styles.scss [hunk 1]
```diff
@@ -44,6 +44,11 @@ body {
   padding: 0 0 $padding-page;
 }
 
+.App-content > div {
```
Comment (muffinresearch) [inline src/amo/components/App/styles.scss]: Thinking a dedicated wrapper would be a plan, to save the possibility of this being brittle in the future.

**Comment by muffinresearch** [comment]
@pwalm here's a quick example of limiting the content width does this look to match what you expected?

**Comment by tofumatt** [comment]
What happens here with the review window modal? Does the blue fill the page or just the content's max-width?

File: src/amo/components/App/styles.scss (hunk 1 above)
Comment (tofumatt) [reply src/amo/components/App/styles.scss]: Yeah this should be changed I'd say before we merge, it should be easy, right?

**Comment by muffinresearch** [comment]
I'm also trying some other options I'm not sure the full-width header/footer are that nice:

Here's an example with just a white background:

full-screen-white-background

**Comment by muffinresearch** [comment]
And with the photon background colour: 

add-ons_for_firefox

**Comment by muffinresearch** [comment]
Hmm going back to the original layout as per this PR, it's better than these last two actually.

**Comment by willdurand** [comment]
I also prefer the full-width header/footer.

**Comment by pwalm** [comment]
full-width header/footer ftw! Looks great. :D

**Comment by muffinresearch** [comment]
Ok, I'll fix the wrapper and we can try it out for real. Thanks!

**Comment by muffinresearch** [comment]
@tofumatt - good call I've pushed a fix for the modals too:

add-ons_for_firefox

**Pull request merged**